uint8 STATE_FAULT = 1
uint8 STATE_OPERATION_ENABLED = 2
uint8 STATE_SWITCH_ON_DISABLED = 3
uint8 STATE_UNKNOW = 255
uint8 state